packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -327,18 +330,21 @@ export class AzureCommunicationCallAdapter implements CallAdapter {
   public async setCamera(device: VideoDeviceInfo, options?: VideoStreamOptions): Promise<void> {
     return await this.asyncTeeErrorToEventEmitter(async () => {
       await this.handlers.onSelectCamera(device, options);
+      this.selectedCameraChanged();
     });
   }
 
   public async setMicrophone(device: AudioDeviceInfo): Promise<void> {
     return await this.asyncTeeErrorToEventEmitter(async () => {
       await this.handlers.onSelectMicrophone(device);
+      this.selectedMicrophoneChanged();
```

Comment:
Looks like this would be called even when the device hasn't actually changed?
For example, if someone calls this function with the old device again?